index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * This strategy is the simplest example of how to get started with the rg-bot package.
+ * The Bot will run around and gather Poppies until it has 4 in its inventory.
+ *
+ * @param {RGBot} bot
+ */
+function configureBot(bot) {
+
+    bot.setDebug(true);
+
+    // This is our main loop. The Bot will invoke this on spawn.
+    // goal: collect 4 Poppies
+    async function startGathering() {
+
+        // Check the Bot's inventory - if it has less than 4 Poppies
+        // then it needs to find and gather one
+        while (bot.getInventoryItemQuantity('Poppy') < 4) {
```

Comment:
can we change this value so that it farms enough poppy flowers to get 100 pts 

I would like to point to these samples as the first 'system' bots and need them to be somewhat capable of winning 🗡️

Reply:
We could but the bot wanders around _a lot_ to find anything beyond the first five or so poppies, so I wouldn't recommend using this as an actual strategy, just as an example of how to use the rg-bot library.

Reply:
let him wander, better than him just doing nothing

Reply:
Increased the count to 100